ci-operator/step-registry/openshift-qe/stress-ng/openshift-qe-stress-ng-chain.yaml

```diff
@@ -0,0 +1,7 @@
+
+chain:
+  as: openshift-qe-stress-ng
+  steps:
+  - ref: openshift-qe-stress-ng
+  documentation: |-
+    This workflow executes stress-ng on a target system
```

Comment:
stressng has many load params, lets mention this specific script only runs the CPU load.